ambry-tools/src/test/java/com.github.ambry/store/StoreCopierTest.java

```diff
@@ -161,15 +161,16 @@ private void setupTestState() throws IOException, StoreException {
     src.start();
     try {
       deletedId = new MockId("deletedId");
-      addMessage(src, deletedId, Utils.Infinite_Time, false, deletedId.getAccountId(), deletedId.getContainerId(),
```

Comment:
why was this changed?

Reply:
Keep it same with before. https://github.com/linkedin/ambry/commit/dd1706531d573047ff99ce60132b0166809a2bcb#diff-744abfbfdb5da3ec75af6f62141777abL166
(Let me do one more change to make it exactly same with this.)

deleteId here is StoreKey, a cast need if we want to deleteId.getAccount().

Reply:
fixed.